conf/waivers

```diff
@@ -0,0 +1,98 @@
+# requires running firewalld (firewall-cmd) and NetworkManager,
+# which are not available in their final form in the Anaconda environment
+# - see https://github.com/ComplianceAsCode/content/issues/9746
+/hardening/anaconda(/with-gui)?/[^/]+/firewalld_sshd_port_enabled
+    rhel >= 8
+
+# https://github.com/OpenSCAP/openscap/issues/1880
+# needs to be remediated more than once due to rule ordering issues
+/hardening/oscap(/with-gui)?/[^/]+/configure_bashrc_exec_tmux
+/hardening/oscap(/with-gui)?/[^/]+/no_tmux_in_shells
+/hardening/oscap(/with-gui)?/[^/]+/configure_usbguard_auditbackend
+    rhel >= 8
+/hardening/oscap(/with-gui)?/[^/]+/configure_bashrc_tmux
+/hardening/oscap(/with-gui)?/stig(_gui)?/postfix_prevent_unrestricted_relay
+    rhel == 8
+
+# rule ordering issue - accounts_password_pam_retry is checked first and passes,
+# and a later enable_authselect remediation breaks it
+# - see https://github.com/OpenSCAP/openscap/issues/1880
+/hardening/.+/accounts_password_pam_retry
+    rhel >= 8
+
+# https://github.com/ComplianceAsCode/content/pull/10499
+# these are missing the required partitions, making the oscap Anaconda addon
+# error out during remediaton
+/hardening/anaconda/cis_(server|workstation)_l1
+    status == 'error' and note.startswith('RuntimeError: installation failed')
+
+# bz1828871, won't be fixed on rhel7
+# needs to be remediated more than once due to rule ordering issues
+/hardening/anaconda(/with-gui)?/[^/]+/postfix_network_listening_disabled
+    rhel == 7
+
+# bz1778661 abrt package
+# - seems to be less predictable on new upstream content
+/hardening/anaconda(/with-gui)?/e8/rpm_verify_(ownership|permissions)
+    Match(rhel == 7, sometimes=True)
+
+# bz1825810 or maybe bz1929805
+# can be reproduced mostly reliably (95%) both via anaconda and oscap CLI,
+# but apparently we don't really care about old releases
+/hardening/[^/]+/with-gui/stig_gui/sysctl_net_ipv4_ip_forward
+    Match(rhel == 7, sometimes=True)
+
+# https://github.com/ComplianceAsCode/content/issues/10424
+/hardening/anaconda(/with-gui)?/[^/]+/service_nftables_disabled
+    ssg >= '0.1.68'
+
+# caused by us not caring whether the VM has it installed,
+# - remediation should fix it, but doesn't -- possibly caused by
+#   https://github.com/RHSecurityCompliance/contest/issues/15
+/hardening/oscap(/with-gui)?/[^/]+/package_scap-security-guide_installed
+    ssg >= '0.1.68'
```

Comment:
Hmm, it's true that this might be upstream infra related, rather than content version related. I've updated https://github.com/RHSecurityCompliance/contest/issues/15, but this will need more debugging as it was failing with upstream even before the COPR repo URLs. So `sometimes=True` is a good idea.